src/utils/emlTemplate.j2

```diff
@@ -83,12 +89,38 @@
                 </boundingCoordinates>
             </geographicCoverage>
         </coverage>
+    <contact>
+        <individualName>
+            <givenName>{{ contact.givenNames}}</givenName>
+            <surName>{{ contact.surName}}</surName>
```

Comment:
Interesting that it is contact.lastName on line 25/29 above, but contact.surName here

Reply:
Good catch, I think that's actually supposed to be `contact.lastName`.